tests/tests-node.js

```diff
@@ -29,7 +29,7 @@ browser = false;
 const TIMEOUT = 1000;
 const LONG_TIMEOUT = 45000;
 const EXPECTED_MAJOR = 1;
-const EXPECTED_MINOR = 1;
+const EXPECTED_MINOR = 3;
 const EXPECTED_PATCH = 0;
 
 describe('get_version', function () {
```

Comment:
maybe we should use camelCase here for consistency ?

Reply:
I tried to keep this similar to the firmware/app so it is easier to track the functionality in C later.
We can refactor if it is really necessary.

Reply:
Sorry, that comment was with respect to the INS constants.
I renamed these ones.